corehq/apps/accounting/forms.py

```diff
@@ -566,7 +566,7 @@ def clean(self):
             and start_date > self.cleaned_data['end_date']):
             raise ValidationError("End date must be after start date.")
 
-        if self.cleaned_data['end_date'] <= datetime.date.today():
+        if self.cleaned_data['end_date'] and self.cleaned_data['end_date'] <= datetime.date.today():
```

Comment:
that's a good point. `end_date` would technically be in the local timezone and `datetime.date.today()` would then be in server time?